src/elements/content-sidebar/activity-feed/comment-form/CommentForm.js

```diff
@@ -152,10 +169,12 @@ class CommentForm extends React.Component<Props, State> {
                             contactsLoaded={contactsLoaded}
                             editorState={commentEditorState}
                             hideLabel
+                            timeStampedCommentsEnabled={allowVideoTimeStamps}
                             isDisabled={isDisabled}
                             isRequired={isOpen}
                             name="commentText"
                             label={formatMessage(messages.commentLabel)}
+                            timeStampLabel={formatMessage(messages.commentTimestampLabel)}
```

Comment:
similar comment here about the casing of `timestamp` vs `timeStamp`. we should be consistent across this PR. isn't timestamp one word instead of two?

Reply:
It can be either one word two but it's more commonly used as oneword